code/game/machinery/computer/camera.dm

```diff
@@ -100,12 +104,16 @@
 			user.unset_machine()
 			return 0
 
+		playsound(src, "terminal_type", 25, 0)
+		playsound(src, 'sound/machines/terminal_prompt_confirm.ogg', 25, 0)
 		if(isAI(user))
 			var/mob/living/silicon/ai/A = user
 			A.eyeobj.setLoc(get_turf(C))
 			A.client.eye = A.eyeobj
 		else
 			user.reset_perspective(C)
+			user.overlay_fullscreen("flash", /obj/screen/fullscreen/flash/noise)

```

Comment:
Doesn't seem really in the scope of PR, but I'll do it. I blame you if oranges gets angry at me for not atomizing again.